tools/nodejs/binding.gyp

```diff
@@ -63,12 +63,6 @@
                         ]
                     },
                 ],  # OS=="win"
-                [
-                    'target_arch=="arm64"',
```

Comment:
It turned out this stuff is already specified in https://github.com/nodejs/node/blob/main/common.gypi#L535-L542